### PR TITLE
Add status filter in backlog Overview

### PR DESCRIPTION
**Pull request by fernandog**
Add status filter in backlog Overview

image

### CODE REVIEW

**Comment by fernandog** [comment]
@wimpyrbx fixed the spaces

File: medusa/server/api/v2/config.py [hunk 1]
```diff
@@ -131,7 +131,8 @@ def get(self, query=''):
                 }
             },
             'selectedRootIndex': int(app.SELECTED_ROOT) if app.SELECTED_ROOT else None,
-            'backlogPeriod': app.BACKLOG_PERIOD
+            'backlogPeriod': app.BACKLOG_PERIOD,
```
Comment (OmgImAlexis) [inline medusa/server/api/v2/config.py]: Would it not be better to use something like this to keep the backlog stuff together?
```
'backlog': {
    'period': app.BACKLOG_PERIOD,
    'status': app. BACKLOG_STATUS
}

File: static/js/manage/backlog-overview.js [hunk 2]
```diff
@@ -70,6 +70,17 @@ MEDUSA.manage.backlogOverview = function() {
...
     });
 
+    $('#backlog_status').on('change', function() {
+        api.patch('config', {
+            backlogStatus: $(this).val()
+        }).then(function(response) {
+            log.info(response);
+            window.location.reload();
+        }).catch(function(err) {
+            log.info(err);
```
Comment (OmgImAlexis) [inline static/js/manage/backlog-overview.js]: Catch should be using `log.error()`.

File: tests/apiv2/test_config.py [hunk 3]
```diff
@@ -124,8 +124,10 @@ def config(monkeypatch, app_config):
             }
         },
         'selectedRootIndex': app.SELECTED_ROOT,
-        'backlogPeriod': app.BACKLOG_PERIOD,
-        'backlogStatus': app.BACKLOG_STATUS
+        'backlogbacklogOverview': {
```
Comment (OmgImAlexis) [inline tests/apiv2/test_config.py]: backlogbacklogOverview? haha

**Pull request merged**